tests/acceptance/features/apiAntivirus/antivirusFileSize.feature

```diff
@@ -42,6 +42,23 @@ Feature: Antivirus file size
       | old              |
       | new              |
 
+  Scenario: Files smaller than the upload threshold are checked for viruses when using chunking (use async move to upload)
+    Given parameter "av_max_file_size" of app "files_antivirus" has been set to "100"
+    And using new DAV path
+    And the administrator has enabled async operations
+    When user "user0" uploads the following chunks asynchronously to "/myChunkedFile.txt" with new chunking and using the WebDAV API
+      | 1 | X5O!P%@AP[4\PZX54(P^)7C |
+      | 2 | C)7}$EICAR-STANDARD-ANT |
+      | 3 | IVIRUS-TEST-FILE!$H+H*  |
+    Then the HTTP status code should be "202"
```

Comment:
Why do we expect 202?

Reply:
see https://github.com/owncloud/core/pull/32414#issuecomment-416892796
after sending the MOVE we don't know the result yet

Reply:
see https://github.com/owncloud/core/pull/32414#issuecomment-416892796